open-craft/blockstore
# [SE-986] Move ElasticSearch from Blockstore to LabXchange

## Description

This PR reverts !20 to remove the ElasticSearch api from blockstore. The search api is now [here](https://gitlab.com/opencraft/client/LabXchange/labxchange-dev/merge_requests/134).

## Author Comments, Concerns, and Open Questions

n/a

## Test Instructions

Review the code change manually to see if any code is being removed which is unrelated to search.
